src/js/components/VoterGuide/PledgeToVoteStatusBar.jsx

```diff
@@ -39,82 +39,83 @@ export default class PledgeToVoteStatusBar extends Component {
   onVoterGuideStoreChange () {
+    const { organization } = this.state;
     this.setState({
-      voter_guide: VoterGuideStore.getVoterGuideForOrganizationIdAndElection(this.state.organization.organization_we_vote_id, VoterStore.election_id()),
+      voterGuide: VoterGuideStore.getVoterGuideForOrganizationIdAndElection(organization.organization_we_vote_id, VoterStore.election_id()),
     });
   }
 
   render () {
     renderLog(__filename);
-    const turned_off = true; // We don't want to use the status bar yet
-    if (turned_off) {
+    const turnedOff = true; // We don't want to use the status bar yet
+    if (turnedOff) {
       return null;
     }
 
-    let number_of_supporters_goal = 0;
-    let number_of_supporters = 0;
-    let percent_complete = 0;
-    let voter_has_pledged = false;
-    if (this.state.voter_guide) {
-      // console.log("PledgeToSupportOrganizationStatusBar this.state.voter_guide: ", this.state.voter_guide);
-      number_of_supporters = this.state.voter_guide.pledge_count || 0;
-      number_of_supporters_goal = this.state.voter_guide.pledge_goal || 0;
+    let numberOfSupportersGoal = 0;
+    let numberOfSupporters = 0;
+    let percentComplete = 0;
+    let voterHasPledged = false;
+    if (this.state.voterGuide) {
+      // console.log("PledgeToSupportOrganizationStatusBar this.state.voterGuide: ", this.state.voterGuide);
+      numberOfSupporters = this.state.voterGuide.pledge_count || 0;
+      numberOfSupportersGoal = this.state.voterGuide.pledge_goal || 0;
 
       // So we can demo
-      number_of_supporters += 10;
-      number_of_supporters_goal = 100;
+      numberOfSupporters += 10;
+      numberOfSupportersGoal = 100;
 
-      if (number_of_supporters_goal !== 0) {
-        const percent_complete_ratio = number_of_supporters / number_of_supporters_goal;
-        const percent_complete_raw = percent_complete_ratio * 100;
-        percent_complete = Math.round(percent_complete_raw);
-        // console.log("percent_complete_raw: ", percent_complete_raw, "percent_complete: ", percent_complete);
-        if (percent_complete < 30) {
+      if (numberOfSupportersGoal !== 0) {
+        const percentCompleteRatio = numberOfSupporters / numberOfSupportersGoal;
+        const percentCompleteRaw = percentCompleteRatio * 100;
+        percentComplete = Math.round(percentCompleteRaw);
+        // console.log("percentCompleteRaw: ", percentCompleteRaw, "percentComplete: ", percentComplete);
+        if (percentComplete < 30) {
           // Increase it to the minimum width, so there is room for the label
-          percent_complete = 30;
+          percentComplete = 30;
         }
       }
-      voter_has_pledged = this.state.voter_guide.voter_has_pledged;
-      // console.log("PledgeToSupportOrganizationStatusBar voter_has_pledged:", voter_has_pledged, ", voter_guide_we_vote_id: ", this.state.voter_guide.we_vote_id);
+      ({ voter_has_pledged: voterHasPledged } = this.state.voterGuide.voter_has_pledged);
```

Comment:
Instead of trying to deconstruct from `this.state.voterGuide` I think it is better to just assign this like this:
`voterHasPledged = this.state.voterGuide.voter_has_pledged`